imsim/opsim_meta.py

```diff
@@ -33,6 +33,30 @@ def get_opsim_md(config, base):
     return opsim_md
 
 
+def opsim_fallback_values(config, base, required_fields):
+    """Loads required fields (specified by a dict where keys are field names
+    and values are the types of the fields) from opsim meta if not specified in config.
+
+    Currently only the type of an Angle will will have an inpact on the processed fields.
+    Angles are assumed to be degree values in opsim meta.
+    """
+    missing_values = set(required_fields) - set(config)
+    if not missing_values:
+        return {}
+    opsim_meta = galsim.config.GetInputObj(
+        "opsim_meta_dict", config, base, "opsim_meta_dict"
+    )
```

Comment:
I actually agree: its a bad idea to introduce such a backdoor.
I currently use it to grab alt/az, exptime and rottelpos.
Is there a way to get those values without opsim meta?
If not: it is always quite some ceremony to extract / convert fields from opsim meta and I have to load the values mentioned above in stamp.py and photon_ops.py.
What would you think about renaming `opsim_fallback_values` to `load_opsim_values` and only request the values which I really need? Any other missing values would raise an error. I think this way it would not be a backdoor anymore.

Reply:
Yes, just make them required.  And users can use OpsimMeta types in the config.